File: cpp/bc1985.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	int fVal, qtd, nPe;
	float total = 0.0;
	
	cin >> nPe;
	
	for(int i = 0; i < nPe; i++)
	{
		cin >> fVal >> qtd;
		
		switch(fVal)
		{
			case 1001: total += qtd*1.50; break;
			case 1002: total += qtd*2.50; break;
			case 1003: total += qtd*3.50; break;
			case 1004: total += qtd*4.50; break;
			case 1005: total += qtd*5.50; break;
		}
	}
	
	cout << fixed << setprecision(2) << total << endl;
}

File: python/bc1985.py
nPe = int(input())
vec = {"1001":1.50, "1002":2.50, "1003":3.50, "1004":4.50, "1005":5.50}
total = 0.0

for c in range(0, nPe):
    cVal = input().split(" ")

    fVal = cVal[0]
    qtd = int(cVal[1])

    total += vec[fVal] * qtd

print(f"{total:.2f}")
